pkg/agent/testdata/AKSUbuntu2204+cgroupv2/CustomData

```diff
@@ -0,0 +1,369 @@
+
+
+
+
+- path: /etc/apt/apt.conf.d/99periodic
+  permissions: "0644"
+  owner: root
+  content: |
+    APT::Periodic::Update-Package-Lists "0";
+    APT::Periodic::Download-Upgradeable-Packages "0";
+    APT::Periodic::AutocleanInterval "0";
+    APT::Periodic::Unattended-Upgrade "0";
+
+
+- path: /etc/logrotate.d/rsyslog-warn
+  permissions: "0644"
+  owner: root
+  content: |
+    /var/log/warn
+    {
+            rotate 4
+            weekly
+            missingok
+            notifempty
+            compress
+            delaycompress
+            sharedscripts
+            postrotate
+                    /usr/lib/rsyslog/rsyslog-rotate
+            endscript
+    }
+- path: /etc/kubernetes/sp.txt
+  permissions: "0600"
+  encoding: base64
+  owner: root
+  content: |
+    U2VjcmV0CiNFT0Y=
+
+
+
+
+
+- path: /etc/systemd/system/sync-tunnel-logs.service
+  permissions: "0644"
+  owner: root
+  content: |
+    [Unit]
+    Description=Syncs AKS pod log symlinks so that WALinuxAgent can include aks-link/konnectivity/tunnelfront logs.
+    After=containerd.service
+
+    [Service]
+    ExecStart=/opt/azure/containers/sync-tunnel-logs.sh
+    Restart=always
+
+    [Install]
+    WantedBy=multi-user.target
+
+- path: /opt/azure/containers/sync-tunnel-logs.sh
+  permissions: "0744"
+  encoding: gzip
+  owner: root
+  content: !!binary |
+    H4sIAAAAAAAA/5RUYW/bNhT8zl9xdYy2yUbLSbF9SJBiW+wVAbxksL0NQ1EEtPQkEaJIjaTsavP++0DKdtwmHVB9sp4f7927O+rkBZKV1MlKuJKxxfzmOlkLmyhTJKnRXkhN1rHJYvlYF3+3lhJRuaQxmWPsBD9ZqQtIjfsFt6SEpwwBEGthHXOmtSkhIZ8mxoUGEo7CuVvtvFAKUhsv8457Y5SDzOFL6l5ZQi2dC9C5NXUoQtaiICZzvH+P4T+3k39xfY1BLWzgOcCHD1ehTTMgNXUtdAa+3sNvhPR4m2S0TnSrFC7evjzHdotM55A7Jrz7lAwj5QhfBScazwvy4AaTX6vi8nJm0urycilrMq2/fjMe91OeWZ3lMsiyII9CmdUqrG4aL412cAa+FD4QaQrSKKxYORituiiMMoXDhrAR2jNXmsaDO9BHH4AeC4FprLAT/BEWyI3FM47DG9BH6XzUGi/AMwwX8xs8CkxpacA1BgEnmmRs37M/OxqNBgzYlFLR5yCZYQDgFFGD8+/iywExHsuMpv2YQXgZDXb6/CIqiktn5LzUIgiETFpKvbFdyI82HkJZElmHxpIj7VldZdKCNxhOFssosxfWQ2Al0qqwptUZGmtScnH7VJHQaJte2RjAjBSFaIfYRzPYCUqxJv3KY0WkUZtM5pKycBUuUJrWuhGWpXQQSpmNQxuhLQWV4VutSQU0dtJPEbknCwFLLnAbsV46b1vaKZZLneELNxHcdw0hB1dSVw7n4HUtNb45vxiD9+TZXvI334/HLKiKl72kuhVKdXCdTgNb1MKnZbA1MjM6JRYMXv52dzedPczu3z38fDubhk2Hr/eZ5O8wCAYnP7wWlYs0tpXRmlIv19J3237l3BrtT/nZQ9WuiLvOeaofzkbKFIPTfTJ622dSV4HD8LOxIR/9d0tp8Fn+pCGanMQFWQ8V3e4/U4frGwMatjdaemNjgtNS6IIcO77l/C/wGtyC0y4E36aWhKc+zttdxEPeMLmdT2+W9/M/Mf19erdEYHOIeyocYbgTLlbCs9fqOUWwxbGCX2g50vW5jtPDqEcSPbsjFvvnZj79cTk9O33yB57acvDiyXNszmB4ECXpj4TSYpn8z/mrq0/K5ES6K4WfvbH/BQAA//+kjvCjvAYAAA==
+
+
+- path: /etc/systemd/system/kubelet.service.d/10-containerd.conf
+  permissions: "0644"
+  encoding: gzip
+  owner: root
+  content: !!binary |
+    H4sIAAAAAAAA/2yPwcrCMBCE732K0nsa/sN/EXKoGkUsFWw9iRRJFwmaTd1sir69iCgVvM0M3zDMvgYarIFDonGw5NEBssrWu6kuddPONlVTrCq9nbeLsljWSgjjkY8WgQRFZOtAETjPkIp3IAiuEQKLp/GR1d+/S38UBWDXe4usItrbREpJEeUH60YyD96cRwvmRD72QclwDwwuDxdr4It/3cqSRwAAAP//1Kx0RuEAAAA=
+- path: /etc/systemd/system/kubelet.service.d/10-cgroupv2.conf
```

Comment:
@UtheMan and therefore, if you look at the customdata generated for 2204 test case, you ll see that it has the 10-cgroupv2.conf file but other test cases dont.